### PR TITLE
fix dependency problems

### PR DESCRIPTION
**Pull request by zimmerml**
fix dependency problems leading to a NoSuchMethodError on windows machines

### CODE REVIEW

File: org.opentosca.container.api/pom.xml [hunk 1]
```diff
@@ -1,153 +1,159 @@
...
+	</parent>
 
-  <artifactId>org.opentosca.container.api</artifactId>
+	<artifactId>org.opentosca.container.api</artifactId>
 
-  <properties>
-    <resteasy.version>3.1.4.Final</resteasy.version>
-    <spring.version>4.3.18.RELEASE</spring.version>
-  </properties>
+	<properties>
```
Comment (nyuuyn) [inline org.opentosca.container.api/pom.xml:15]: Can we move this up into the main pom.xml?

**Review by wederbn — APPROVED** [review]
LGTM (except the reformatting)

File: org.opentosca.container.api/pom.xml [hunk 2]
```diff
@@ -1,153 +1,159 @@
...
+		</dependency>
 
-<!--    <dependency>-->
-<!--      <groupId>org.jboss.resteasy</groupId>-->
-<!--      <artifactId>resteasy-spring</artifactId>-->
-<!--      <version>${resteasy.version}</version>-->
-<!--    </dependency>-->
+		<!-- <dependency> -->
+		<!-- <groupId>org.jboss.resteasy</groupId> -->
+		<!-- <artifactId>resteasy-spring</artifactId> -->
```
Comment (nyuuyn) [inline org.opentosca.container.api/pom.xml:114]: maybe remove it

File: org.opentosca.container.war/pom.xml [hunk 3]
```diff
@@ -609,9 +613,9 @@
 		</dependency>
 
 		<!-- provided by tomcat -->
-		<!-- We're using 3.0.1 as opposed to 3.0 (which is what the web.xml requires)
-			because that is the only spec version available from maven-central that's
-			not marked as prerelease. Enforcing 3.1 would put us onto tomcat8 Enforcing
+		<!-- We're using 3.0.1 as opposed to 3.0 (which is what the web.xml requires) 
+			because that is the only spec version available from maven-central that's
```
Comment (nyuuyn) [inline org.opentosca.container.war/pom.xml:617]: This commen still relevant ?

**Review by nyuuyn — APPROVED** [review]
lgtm

File: org.opentosca.container.api/pom.xml (hunk 2 above)
Comment (zimmerml) [reply org.opentosca.container.api/pom.xml]: done

File: org.opentosca.container.war/pom.xml (hunk 3 above)
Comment (zimmerml) [reply org.opentosca.container.war/pom.xml]: no. removed it

**Pull request merged**